### PR TITLE
Adding indian-hc-finder.hackclub.com

### PR DESCRIPTION
**Pull request by MohitTiwariBytes**
#Adding indian-hc-finder.hackclub.com

## Description

I have created a web-app that lists all the indian hackclubs around the your location!

### CODE REVIEW

**Comment by Muirrum** [comment]
What's the difference between this and #1213 ?

File: hackclub.com.yaml [hunk 1]
```diff
@@ -2903,3 +2903,9 @@ takeout:
   - ttl: 600
     type: A
     value: 68.183.252.105
+
+indian-hc-finder:
+  ttl: 600
+  type: TXT
+  values:
+    - vc-domain-verify=indian-hc-finder.hackclub.com,6bc40daab2feffbb2061
```
Comment (Muirrum) [inline hackclub.com.yaml:2911]: This should go in the main _vercel record at the top of the file. See README for more details

Comment (Muirrum) [inline hackclub.com.yaml:2911]: addtl -- this doesn't point to vercel. suggest following CNAME record

```suggestion

indian-hc-finder:
  - ttl: 600
    type: CNAME
    value: cname.vercel-dns.com.
```

**Review by Muirrum — CHANGES_REQUESTED** [review]

**Comment by MohitTiwariBytes** [comment]
Feels a lil weird to say this, but I am really new into this thing, so I probably giving up at this moment so sorry, I'll tell my friend Nimit to do this, thank you so much

**Pull request closed without merging**